docs/rfc/rfc-1.adoc

```diff
@@ -465,7 +465,27 @@ BTC from the wallet to another random wallet and close this wallet.
 
 [[transaction-incentives]]
 === Transaction Incentives
-stub!
+We use two main types of transaction incentives: Unlucky Operators and Escalating Rewards.
+
+[[unlucky-operator]]
+==== Unlucky Operators
+The Unlucky Operator scheme involves using the a source of pseudo-randomness
+(like the next ethereum block hash) to choose who the operator to pay for the
+gas is. If that operator refuses, the remaining operators can sign a message to
```

Comment:
> If that operator refuses
What do we constitute as a refusal? Do we set some time (or no. of blocks) threshold after we treat the operator as refusing to pay? If so, will it be a common governable value, or will it be determined on a case-by-case basis (different value for different usages of the unlucky operator scheme).